aries-site/src/pages/components/table.mdx

```diff
@@ -0,0 +1,240 @@
+  alt="Example demonstrating how to find a record of interest with a Table"
+/>
+
+### Assembling a data set
+Distinct from the user objectives above, tables containing data for analytical 
+purposes deserve distinct treatments as well. While tables are useful for 
+providing initial "discovery" type insights, such as identifying available attributes
+and ranges of values, purpose built tools are often a better choice for conducting 
+in depth analysis. In this scenario, a table's design should be optimized for 
+assembling the data set for analysis and easy portability of that data into the 
+user's preferred analytical tool.
+
+- Provide [filter controls](#searching-and-filtering-tables).
+- Filter control input types should be chosen to best fit the data type and breadth of values.
+- Provide export capability for users to consume the data in their analytical 
+tool of choice.
+- Provide aggregate measures in the [table footer](#column-summaries-and-aggregation).
+
+<!--
+## Guidelines
+-->
+
+<!--
+### How many columns?
+The number of columns and rows should be guided by the task the user is setting 
+out to accomplish.
+
+Generally, if there are more than 5-7 columns in the table, it is a sign that 
+a single table is trying to solve too many user problems at once. In that case, 
+consider:
+- Are there too many user persona types using this view? If so, are there opportunities 
+to personalize the view by role?
+- What are the user interactions leading up to the table view? Are there opportunities 
+to improve navigation and build intentful user flows?
+- Perhaps, it is unclear as to what the user problem the table is attempting 
+to solve? A good opportunity for speaking to and learning from end users!
+
+**The Exception**<br />
+In an analytical context, where the goal is to assemble a data set and identify patterns 
+within it, a larger number of columns may be neccessary.
+
+In this context, a Table should facilitate the assembly of the data set and make the set 
+accessible in a tool specifically designed for analysis.
+-->
+
+## Table behaviors & actions
+
+### Sorting
+Column headers behave as a button and perform the sort action. A caret icon indicates 
+the directionality and to which column the sort is currently applied.
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/table/TableSortable.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
+  height={{ min: 'small' }}
+>
+  <TableSortable />
+</Example>
+
+### Searching and filtering tables
+Complimenatary elements such as search or filters should be guided by the task the user 
+is setting out to accomplish.
+- Looking for a specific record? --> Search.
+- Trying to narrow records down to a set of shared characteristics? --> Filter.
+- Not quite certain for what I am looking, but I know its general characteristics? --> Filter.
+
+[See Searching & Filtering]() for greater detail.
```

Comment:
Wondering if these should be broken out into individual links, since each will live on their own page.

Reply:
Agree.

Reduced to only filtering for now.